contrib/jenkins_tests/test.sh

```diff
@@ -8,6 +8,11 @@ if [ $(test -d ${install_dir} >/dev/null 2>&1 || echo $?) ]; then
 	exit 1
 fi
 
+if [ $(command -v ibdev2netdev >/dev/null 2>&1 || echo $?) ]; then
```

Comment:
this cannot be here i removed it in my commit

Reply:
?

Reply:
I don't see it there